analysis_options.yaml

```diff
@@ -2,7 +2,8 @@ include: package:pedantic/analysis_options.yaml
 
 analyzer:
   exclude:
-    - test/resources/**
+    - test/resources/*
+    - test/resources/unused_files_analyzer/**
     - test/resources/unused_files_analyzer/generated/**/**
```

Comment:
@incendial you can use only one `    - test/resources/unused_files_analyzer/**`

Reply:
Removed.